app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml

```diff
@@ -4,6 +4,8 @@
       .browse-ontology-title-bar
         %a.browse-ontology-title{:href => "/ontologies/#{ontology[:acronym]}", data: {'turbo-frame': '_top'}}
           = ontology[:name]+" ("+ontology[:acronym]+")"
+          - ontology_status = status_string(ontology)
```

Comment:
show this only if you logged in and are admin

Reply:
okey

Reply:
Can be simplified to this `session[:user]&.admin?`

Reply:
okey